import/src/schema/schema_parse.rs

```diff
@@ -0,0 +1,120 @@
+use serde::de::{Deserialize, Deserializer};
+use serde::ser::{Serialize, Serializer};
+use serde::*;
+use std::collections::{HashMap, HashSet};
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Schema {
+    pub org_id: String,
+    pub bucket_id: String,
+    pub measurements: HashMap<String, Measurement>,
```

Comment:
Something you might consider is using the `Schema` here instead of `Measurement`:

```suggestion
    pub measurements: HashMap<String, crate::schema::Schema>,
```

Though that might require some serialization / deserialization dancing

Reply:
i am working against a pre-existing schema so this would be incompatible unless i'm missing something

Reply:
Perhaps you mean that some other tool is writing out JSON that iox needs to interpret? If so, then yes you would have to provide a way to parse that JSON and convert it to a `Schema` -- you will likely have to do eventually to create parquet files so perhaps that would be the right time to consider unificiation

Reply:
i'm guessing perhaps you meant the `schema` crate in the root of IOx rather than `crate::schema` which from this POV would mean `import::schema`? i think that is addressed in the other comment about why I didn't use that merging code, although your point about needing to make parquet files is a good one if so.

i can't really picture what that code is going to look like until i get closer to implementing it, there certainly needs to be some sort of adapter from this schema-like thing to the main IOx one. i think addressing your other comment that suggests renaming this would go a long way towards addressing this.

Reply:
i think this is resolved now by the rename and the comments.